lms/static/scripts/test-util/accessibility.js

```diff
@@ -0,0 +1,87 @@
+import { run } from 'axe-core';
```

Comment:
This module was copied from the client repo. I think we are or soon will be at a point where it could make sense to extract a `@hypothesis/ui-test-utils` package out of the `test-util` directory here / in the client and share this code.

Reply:
There are a number of util like things that would be nice to share....rpc files come to mind. I wonder if we can think broadly with any shared code changes?